contrib/download-frozen-image-v2.sh

```diff
@@ -63,8 +63,10 @@ fetch_blob() {
 			-D-
 	)"
 	curlHeaders="$(echo "$curlHeaders" | tr -d '\r')"
-	if echo "$curlHeaders" | grep -qE "^HTTP/[0-9].[0-9] 3"; then
+#	if echo "$curlHeaders" | grep -qE "^HTTP/[0-9].[0-9] 3"; then
+	if grep -qE "^HTTP/[0-9].[0-9] 3" <<<"$curlHeaders"; then
```

Comment:
I haven't seen any failures yet with this vs line above. I also can't explain why 🤕  

I thought $curlHeaders might be piping nil for some reason, but I ran a `echo $curlHeaders | cat` and even with this failed that succeeded 

Also made an infinite loop that kept pulling everything but couldn't replicate this after 10minutes on my s390x vm running the same debian + bash versions.